wallet/thread/merges.go

```diff
@@ -1,15 +1,19 @@
 package thread
 
 import (
+	"bytes"
 	"github.com/golang/protobuf/proto"
 	"github.com/textileio/textile-go/pb"
 	"github.com/textileio/textile-go/repo"
+	"github.com/textileio/textile-go/util"
+	"gx/ipfs/QmZoWKhxUmZ2seW4BzX6fJkNR8hh9PsGModr7q171yq2SS/go-libp2p-peer"
 	mh "gx/ipfs/QmZyZDi491cCNTLfAhwcaDii2Kg4pwKRkhqQzURGDvY6ua/go-multihash"
+	"sort"
 	"time"
 )
 
-// Merge adds an outgoing merge block, will only post to network if post is true
-func (t *Thread) Merge(head string, post bool) (mh.Multihash, error) {
+// Merge adds a merge block, which are kept local until subsequent updates, avoiding possibly endless echoes
+func (t *Thread) Merge(head string) (mh.Multihash, error) {
```

Comment:
do we hit similar problems as the repeater if we just have a crowded and busy room?

Reply:
I think we might actually be able to post these now that they are hashed identical on each peer, but wanted to see how the current setup works... it seems to be sufficient and is def less noisy